sw/otbn/crypto/tests/generated/BUILD

```
@@ -33,3 +34,55 @@ py_binary(
     )
     for i in range(10)
 ]
+
+# Run randomly generated tests for SHA3.
+py_binary(
+    name = "sha3_testgen",
+    srcs = ["sha3_testgen.py"],
+    imports = [
+        "../../../../../hw/ip/otbn/util",
+    ],
+    deps = [
+        "//hw/ip/otbn/util/shared:testgen",
+        requirement("pycryptodome"),
+    ],
+)
+
+[
+    otbn_autogen_sim_test(
+        name = "sha3_test" + str(i),
+        srcs = ["sha3_py_test.s"],
+        seed = i,
+        testgen = ":sha3_testgen",
+        deps = [
+            "//sw/otbn/crypto:sha3-shake",
+        ],
+    )
+    for i in range(100)
```

nit: let's keep it to `10`, since this will run in CI for every PR in the future.
```suggestion
    for i in range(10)
```